derp/derp_test.go

```diff
@@ -1598,3 +1599,28 @@ func TestServerRepliesToPing(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPerClientSendQueueDepth(t *testing.T) {
+	envKey := "TS_DEBUG_DERP_PER_CLIENT_SEND_QUEUE_DEPTH"
+
+	testCases := []struct {
+		envVal string
+		want   int
+	}{
+		// Empty case, envknob treats empty as missing also.
+		{
+			"", defaultPerClientSendQueueDepth,
+		},
+		{
+			"64", 64,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(fmt.Sprintf("Testing Env Value: %s", tc.envVal), func(t *testing.T) {
```

Comment:
try to avoid punctuation and spaces in subtest names, as it makes it harder to run them in the CLI later by hand.

Maybe `cmp.Or(tt.envVal, "empty")` as the Run name?